sdks/java/extensions/sql/zetasql/src/main/java/org/apache/beam/sdk/extensions/sql/zetasql/translation/ExpressionConverter.java

```diff
@@ -805,6 +805,9 @@ private RexNode convertSimpleValueToRexNode(TypeKind kind, Value value) {
         if (wrapperFun == null) {
           ret = rexBuilder().makeApproxLiteral(new BigDecimal(val), returnType);
         } else if (BeamBigQuerySqlDialect.DOUBLE_NAN_FUNCTION.equals(wrapperFun)) {
+          // 'https://issues.apache.org/jira/browse/BEAM-10550'
```

Comment:
Usually I will make a TODO here (such that you can easily find it in IntelliJ), like:

```
// TODO[BEAM-10550] Update the temporary workaround below after vendored Calcite version >= 1.23.0
```

Reply:
Done.